### PR TITLE
Added features required for the event-os

### PR DESCRIPTION
**Pull request by meets2tarun**
### Short Description
I have:
- [x] There is a corresponding issue for this pull request. #67 #64 
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Configured the generator branch to add lxqt and added chromium config(all are tested and working) and the required meta package as mentioned in the issue.

### CODE REVIEW

**Comment by xeon-zolt** [comment]
locally tested ?

**Comment by abishekvashok** [comment]
Approvers please pull and test it locally.

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,6 @@ sudo: required
 
 script:
   - ./build.sh
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
```
Comment (abishekvashok) [inline .travis.yml]: You would need this as the meilix PPA should get updated with the new sources.

File: event-metapackage_1.0_all/control [hunk 2]
```diff
@@ -0,0 +1,26 @@
+### Commented entries have reasonable defaults.
+### Uncomment to edit them.
+Source: event-metapackage
+Section: misc
+Priority: extra
+# Homepage: <enter URL here; no default>
+Standards-Version: 3.9.2
+
+Package: event-metapackage
+# Version: <enter version here; defaults to 1.0>
```
Comment (abishekvashok) [inline event-metapackage_1.0_all/control]: Please enter a version.

File: build.sh [hunk 3]
```diff
@@ -92,12 +93,8 @@ dpkg -i meilix-metapackage*.deb
 apt-get install -f
 
 # Install base packages
-apt-get -q -y --purge install --no-install-recommends \
-  lxsession lightdm lightdm-gtk-greeter lxterminal gvfs-backends seahorse \
-  network-manager-gnome xorg dbus-x11 openbox ubiquity-frontend-gtk vlc \
-  xfce4-mixer gstreamer0.10-alsa pulseaudio pavucontrol lxpanel \
- mozilla-plugin-vlc  lubuntu-core jockey-gtk
- 
+apt-get -q -y --purge install lxqt
```
Comment (xeon-zolt) [inline build.sh]: add openbox

**Comment by meets2tarun** [comment]
>i have achieved firefox using webapp
>it would be good if you just go with lxqt only in this pull request

Continue customizing the firefox config using webapp
This PR will fix the issue #67

**Comment by xeon-zolt** [comment]
that is a Epic issue so it should be broken to parts !

File: meilix-default-settings/etc/skel/.mozilla/firefox/*.default/prefs.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+user_pref("extensions.databaseSchema", 19);
+user_pref("extensions.e10s.rollout.blocklist", "{dc572301-7619-498c-a57d-39143191b318};firefox@mega.co.nz");
+user_pref("extensions.e10s.rollout.hasAddon", false);
+user_pref("extensions.e10s.rollout.policy", "50allmpc");
+user_pref("extensions.e10sBlockedByAddons", true);
+user_pref("extensions.enabledAddons", "%7B6e84150a-d526-41f1-a480-a67d3fed910d%7D:1.5.6.1-signed.1-signed,fireforce%40scrt.ch:2.2.1-signed.1-signed,%7B972ce4c6-7e08-4474-a285-3208198ce6fd%7D:53.0.3");
+user_pref("extensions.getAddons.cache.lastUpdate", 1498203354);
+user_pref("extensions.getAddons.databaseSchema", 5);
+user_pref("extensions.hotfix.lastVersion", "20170302.01");
+user_pref("extensions.ieview.installTime", "Thu Jun 01 2017 19:57:00 GMT+0530 (IST)");
```
Comment (abishekvashok) [inline meilix-default-settings/etc/skel/.mozilla/firefox/*.default/prefs.js]: Install time?

**Pull request closed without merging**